docs/core/building/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Build .NET Core from source
+description: Build .NET Core from source
+keywords: .NET, .NET Core, source, build
+author: beleroy
+ms.author: mairaw
+ms.date: 03/29/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 8b49079c-6ede-429a-92d7-ecd2fda1ab0e
+---
+
+# Build .NET Core from source
+
+The ability for anyone to build .NET Core from its source code is important in multiple ways: it makes it easier to port .NET Core to new platforms, it enables contributions and fixes to the product, and it enables the creation of custom versions of .NET.
+This section of the documentation gives guidance to developers who want to build and distribute their own versions of .NET Core.
+
+## How to build from source
```

Comment:
since this header doesn't have any content, should you just move the inner sections up and call them 
Build the CLR from source and Build the CLI from source? This would also make the headings show up in the right side nav while H3s don't